workflows/prognostic_c48_run/tests/test_tendency_prescriber.py

```diff
@@ -0,0 +1,92 @@
+from datetime import timedelta
+import joblib
+import cftime
+import numpy as np
+import xarray as xr
+
+import fv3gfs.util
+from fv3gfs.util.testing import DummyComm
+
+from runtime.tendency_prescriber import (
+    TendencyPrescriberAdapter,
+    TendencyPrescriberConfig,
+)
+
+
+class MockDerivedState:
+    def __init__(self, state: xr.Dataset, time: cftime.DatetimeJulian):
+        self._state = state
+        self._time = time
+
+    @property
+    def time(self) -> cftime.DatetimeJulian:
+        return self._time
+
+    def __getitem__(self, key: str) -> xr.DataArray:
+        return self._state[key]
+
+    def __setitem__(self, key: str, value: xr.DataArray):
+        self._state[key] = value
+
+
+def _get_tendency_ds(time):
```

Comment:
I'd drop the "ds" abbreviation. Using type hints makes it clear what the output type is.

```suggestion
def _get_tendency(time: cftime.DatetimeJulian) -> xr.Dataset:
```